test/testdata/lsp/definitions_rb_and_rbi__3.rbi

```diff
@@ -0,0 +1,10 @@
+# typed: true
+
+# The usage annotation here is kind of a hack: we want the only
+# definition for go-to-definition to exist in the rb file defining
+# MyClass, but the test harness will see this class if we don't
+# say that it's a "use", even though semantically it is a def.
+class MyClass
+    # ^^^^^^^ usage: MyClass
```

Comment:
I guess the better way of phrasing is it that the "usage:" assertion isn't really a "use"; it's a check for "find all references".  So by putting a "usage:" here, we're saying:

1. this shouldn't be found as a def by go-to-definition (it is found as a def prior to this PR)
2. this *should* be found as a reference via "find all references" (as it still is with this PR)

which is the behavior we want, even though the assertion names make it a little awkward.